CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## Developer Changelog
+### August 17, 2020 (DESCW-538)
+- Frontend
+  - Resources view modal
+
```

Comment:
Did you not update the contacts as well?

Reply:
kind of by i was fixing for @mhaswell-bcgov